apps/central/src/components/SchemaCreateModal.vue

```diff
@@ -50,17 +50,22 @@
           </LayoutForm>
           <hr />
           <p>Templates</p>
-          <ButtonAlt
-            @click="
-              schemaName = 'CohortCatalogue';
-              sourceURL = [
-                host + '/public_html/apps/data/datacatalogue',
-                host + '/public_html/apps/data/datacatalogue/Cohorts',
-              ];
-            "
-          >
-            Cohort Catalogue
-          </ButtonAlt>
+          <ul>
+            <li>
+              <ButtonAlt
+                @click="
+                  schemaName = 'CohortCatalogue';
+                  sourceURL = [
+                    host + '/public_html/apps/data/datacatalogue',
```

Comment:
this ties the frontend to specific backend details ( that should probably remain hidden ) , suggest passing the id of a template instead of more low level url

Reply:
I considered this then then rejected this idea. I think it is useful for data managers to understand what is going on here, so they could use this also for other data models.